src/components/partnership/dto/partnership.dto.ts

```diff
@@ -54,15 +52,15 @@ export class Partnership extends IntersectionType(ChangesetAware, Resource) {
   } satisfies ResourceRelationsShape;
   static readonly Parent = import('../../project/dto').then((m) => m.IProject);
 
-  readonly project: ID;
+  readonly project: LinkTo<'Project'>;
 
   @Field(() => IProject)
   declare readonly parent: BaseNode;
 
   @Field()
   readonly agreementStatus: SecuredPartnershipAgreementStatus;
 
-  readonly mou: Secured<LinkTo<'File'>>;
+  readonly mou: Secured<LinkTo<'File'>> | null;
```

Comment:
`Secured`'s are never nullable, but they can wrap nullable things, like this file.
```suggestion
  readonly mou: Secured<LinkTo<'File'> | null>;
```